chore: update @edx/brand dependency to new package @openedx/brand-openedx

ttak-apphelix opened the pull request:
### Description

This PR replaces brand-edx.org with openedx theme and adds it to the package file indirectly via aliasing.

### Supporting information

Issue Link: edx/edx-arch-experiments#943

Link to the PR which is associated with the same issue
https://github.com/edx/configuration/pull/230

ttak-apphelix committed:
chore: update @edx/brand dependency to new package @openedx/brand-openedx

ttak-apphelix committed:
Merge branch 'master' into ttak-apphelix/branding-alias

ttak-apphelix committed:
Merge branch 'master' into ttak-apphelix/branding-alias

kdmccormick reviewed (changes requested):
Thanks for the PR. I'm concerned about the size of the package-lock.json change-- that seems way to high of a line count for such a small package.json change. What command did you run to regenerate package-lock.json?

robrap commented:
@ttak-apphelix: Next steps:
1. Work with @kdmccormick to see if you can get a simpler change set, regarding:
> Thanks for the PR. I'm concerned about the size of the package-lock.json change-- that seems way to high of a line count for such a small package.json change. What command did you run to regenerate package-lock.json?
2. Did we already put QE resources toward testing changes? If not, let's do that, but after seeing if there is a simplification.
3. Once all this had been done, we'll continue to see if we can get 2U reviews. Thanks.

ttak-apphelix committed:
Merge branch 'openedx:master' into ttak-apphelix/branding-alias

ttak-apphelix committed:
Revert package-lock.json

ttak-apphelix committed:
fix: updated package-lock.json

ttak-apphelix commented:
> Thanks for the PR. I'm concerned about the size of the package-lock.json change-- that seems way too high of a line count for such a small package.json change. What command did you run to regenerate package-lock.json?

@kdmccormick  I was deleting node_module folder but by mistake I added package-lock.json file also. I have corrected that change and now it shows minimal changes in package-lock.json file

ttak-apphelix commented:
> @ttak-apphelix: Next steps:
> 
> 1. Work with @kdmccormick to see if you can get a simpler change set, regarding:
> 
> > Thanks for the PR. I'm concerned about the size of the package-lock.json change-- that seems way to high of a line count for such a small package.json change. What command did you run to regenerate package-lock.json?
> 
> 2. Did we already put QE resources toward testing changes? If not, let's do that, but after seeing if there is a simplification.
> 3. Once all this had been done, we'll continue to see if we can get 2U reviews. Thanks.

@robrap 
1. I have corrected the changes and they are minimal in package-lock.json.
2. I have tested myself and ran this branch on sandbox and everything is working as it was earlier.

kdmccormick reviewed (approved):
Thanks, the code looks good to me. I haven't tested, so please make sure some basic smoke-testing has been done before merging.
